jina/resources/fluent.conf

```diff
@@ -13,9 +13,9 @@
 ## match tag=myapp.** and forward and write to file in local
 <match jina.**>
   @type file
-  path /tmp/jina-log
+  path /tmp/jina-log/${context}/log
```

Comment:
this would be changed to `id` so that `logs` are ordered by `id` which will be intended to be offered by the `gateway` in `jinad`